docs/computing/alliancecan.md

```diff
@@ -156,6 +171,59 @@ ssh -Y <user>@graham.computecanada.ca
 By default, this brings you to the login node which **is not** a compute node. 
 This should only be used for viewing images.
 
+### Run Jupyter Notebooks in your local VS Code
+To be able to run Jupyter Notebooks in Visual Studio Code from your local machine, use the following instructions:
```

Comment:
Should we make this explicit about running Jupyter notebooks started on Graham?

For instance, you could just call Jupyter locally to read local / mounted files. There is also a Jupyter section under the Software page that describes the local usage of Jupyter without going into details about Graham.